### ISSUE TITLE
Dissallowing "Re-sharing" makes sharing impossible in Calendar

### ISSUE THREAD

**Issue by mr-gosh** [owncloud/apps#405]
I disallowed my users the "re-sharing" of items the got shared to them - which completely makes sense. BUT - that makes it impossible to share their own calendars any more...

I I shared my calendar before the option got unchecked everything works fine, so I didn't recognize the Problem from the beginning. But I unchecked the sharing of an calendar by accident and wanted to re-share it immediately. But that was impossible.
I had to allow "Re-sharing" again to fix this.

Is the Problem clear?

tl;dr:
sharing calendar = works
dissallow resharing = still works
unsharing a calendar and reactivate the sharing = doesnt work
